src/transfers-history/config.ts

```diff
@@ -0,0 +1,24 @@
+import { ChainId, CHAIN_IDs } from "./adapters/web3/model";
+
+export type SpokePoolConfig = {
+  addr: string;
+  lowerBoundBlockNumber: number;
+};
+
+export type ClientConfig = {
+  web3ProvidersUrls: Record<ChainId, string>;
+  spokePools: Record<ChainId, SpokePoolConfig>;
+};
+
+export const clientConfig: ClientConfig = {
+  web3ProvidersUrls: {},
+  spokePools: {
+    [CHAIN_IDs.ARBITRUM_RINKEBY]: {
+      addr: "0x68306388c266dce735245A0A6DAe6Dd3b727A640",
+      lowerBoundBlockNumber: 9828565,
+    },
+    [CHAIN_IDs.OPTIMISM_KOVAN]: { addr: "0x99EC530a761E68a377593888D9504002Bd191717", lowerBoundBlockNumber: 0 },
+    [CHAIN_IDs.RINKEBY]: { addr: "0x90743806D7A66b37F31FAfd7b3447210aB55640f", lowerBoundBlockNumber: 0 },
+    [CHAIN_IDs.KOVAN]: { addr: "0x73549B5639B04090033c1E77a22eE9Aa44C2eBa0", lowerBoundBlockNumber: 0 },
+  },
```

Comment:
Any reason we wouldn't ask that the caller provide this? Preferably, there's one source of truth for these sorts of addresses, either in the FE, where it's passed to the SDK or the SDK holds these address constants and they are used by the FE where it needs them.

Right now, I think both have a consts file, which could lead to issues where one is updated and the other is not.

Reply:
[I changed](https://github.com/across-protocol/sdk-v2/commit/e97b8ce0b4c82c65dfa6d775cd143dccc82770ac#diff-916d18a80c514fc8c883139e2cd5e3e3bcad780b37da4caf1e47b58293c3476eR27) the config passed to the constructor method to contain the addresses of the SpokePool contracts